content/projects.md

```diff
@@ -122,7 +122,7 @@ protocol.
 #### [TOML](https://github.com/BurntSushi/toml)
 
 An encoder and decoder for [TOML, an ini-like config file
-format](https://github.com/toml/toml). Incidentally, TOML is used to configure
+format](https://github.com/toml-lang/toml). Incidentally, TOML is used to configure
```

Comment:
Thanks! But could you please wrap this to 79 columns inclusive, to be consistent with the surrounding style?

Reply:
Sure!